### DISCUSSION TITLE
Taken down from browser extension stores?

### DISCUSSION THREAD

**Discussion by mattheuswhale** [aclap-dev/video-downloadhelper#111] [Help and Q&A] [answered]
### VDH is taken down from extension stores

In case you're not aware, the extension is taken down from firefox extension store. It was having trouble downloading some m3u8 streams so I was about to check the latest reviews but the store page is not available anymore. I don't know what's going on, since I can't find any discussions/posts related to this in the last 24 hours. For now, I uninstalled this and the companion app for the time being. I've been using the app for years and I hope all gets sorted out soon.

**Comment by paulrouget** [comment, marked as answer]
Thanks. We are working with Mozilla on that: https://github.com/aclap-dev/video-downloadhelper/discussions/109
